ONSdigital/census-rm-ops
# Import QID batch runner through pipenv

# Motivation and Context
Ops tool is currently using copied code, this PR is to proof of concept importing the module from the github source instead of copy pasting the code.

# What has changed
- Removed the copy of the generate qid batch code
- Import the qid batch runner from a github branch with package setup
- Pipenv update

# How to test?
You should be able to install dependencies, test and use the ops tool exactly as before.

# Links
https://github.com/ONSdigital/census-rm-qid-batch-runner/pull/4
https://trello.com/c/GgRSVlOY/795-importing-python-modules-from-github-repos-1-hour
